Library/Formula/tarantool.rb

```diff
@@ -0,0 +1,66 @@
+class Tarantool < Formula
+  desc "In-memory database and Lua application server."
+  homepage "http://tarantool.org"
+  url "http://tarantool.org/dist/master/tarantool-1.6.7-562-g3dbc9ff-src.tar.gz"
+  version "1.6.7-562"
+  sha256 "6aded257e11fa114dc21e8493ff96dfe3f8a692d62e096afb5105cecc179211d"
+  head "https://github.com/tarantool/tarantool.git", :shallow => false
+
+  devel do
+    url "https://github.com/tarantool/tarantool.git", :branch => "1.6", :shallow => false
+    version "1.6.8"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "readline"
+
+  def install
+    args = std_cmake_args
+    args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
+    args << "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
+    if build.devel?
+        args << "-DENABLE_DIST=ON"
+    end
+
+    system "cmake", ".", *args
+    system "make"
+    system "make", "install"
+
+    inreplace bin/"tarantoolctl", %r{\/usr\/local\/etc\/tarantool\/default\/tarantool}, "#{etc}/default/tarantool"
+  end
+
+  def postinstall
+    inreplace etc/"default/tarantool" do |s|
+      local_user = ENV["USER"]
+      s.gsub!(/(pid_file\s*=).*/, "\\1 '#{var}/run/tarantool',")
+      s.gsub!(/(wal_dir\s*=).*/, "\\1 '#{var}/lib/tarantool',")
+      s.gsub!(/(snap_dir\s*=).*/, "\\1 '#{var}/lib/tarantool',")
+      s.gsub!(/(sophia_dir\s*=).*/, "\\1 '#{var}/lib/tarantool',")
+      s.gsub!(/(logger\s*=).*/, "\\1 '#{var}/log/tarantool',")
+      s.gsub!(/(username\s*=).*/, "\\1 '#{local_user}'")
+      s.gsub!(/(instance_dir\s*=).*/, "\\1 '#{etc}/tarantool/instances.enabled'")
+    end
```

Comment:
It's conflicting with this comment
https://github.com/Homebrew/homebrew/pull/47495#discussion_r49264261